edx_proctoring/static/proctoring/js/collections/proctored_exam_onboarding_collection.js

```diff
@@ -0,0 +1,13 @@
+edx = edx || {};
+(function(Backbone) {
+    'use strict';
+
+    edx.instructor_dashboard = edx.instructor_dashboard || {};
+    edx.instructor_dashboard.proctoring = edx.instructor_dashboard.proctoring || {};
+
+    edx.instructor_dashboard.proctoring.ProctoredExamOnboardingCollection = Backbone.Collection.extend({
```

Comment:
#NIT the name of this file and the variable name `ProctoredExamOnboarding` sounds backwards to me. May sound better if its `ProctoredOnboardingExam`?